### PR TITLE
Correct metrics content for LCP to include removed elements

### PR DESCRIPTION
**Pull request by runjak**
Hey 👋 

When reading [web.dev/lcp/](https://web.dev/lcp/) I've noted the aside section pointing to WICG/largest-contentful-paint/issues/41.

My current understanding is that the issue has been closed for a few months and that the decision was made to keep considering removed elements for LCP. Hence the adjustment.

I believe to have signed the CLS accordingly.
I'm a  bit unsure whether an issue should still be created to link it in the commit? Please advise.

### CODE REVIEW

File: src/site/content/en/metrics/lcp/index.md [hunk 1]
```diff
@@ -161,18 +161,7 @@ the DOM as new content becomes available. If any of these new elements is
...
-until the new image loads.
-
-{% Aside %}
-  In the future, elements removed from the DOM may still be considered as LCP
-  candidates. [Research is currently being
-  done](https://github.com/WICG/largest-contentful-paint/issues/41#issuecomment-583589387)
-  to assess the impact of this change. You can follow the metrics
-  [CHANGELOG](http://bit.ly/chrome-speed-metrics-changelog) to stay up-to-date.
-{% endAside %}
+If a page removes an element from the DOM, that element will still be considered.
```
Comment (mihajlija) [inline src/site/content/en/metrics/lcp/index.md:164]:
```suggestion
In order to correctly compute LCP in cases such as image carousels, the LCP algorithm 
considers removed elements. If a page removes an element from the DOM, that element 
will still be considered.
```

**Review by mihajlija — APPROVED** [review]

**Comment by mihajlija** [comment]
Thanks for the update @runjak!

**Comment by philipwalton** [comment]
I think I would phrase this slightly differently. The main information we want to convey here is that an element that was removed from the DOM can continue to be considered the LCP candidate. I think we should lead with that. I don't think the bit about carousels is relevant enough to be the first thing mentioned (I think we can mention it in the aside).

How about something like:

"If an element that is currently the largest contentful element is removed from the viewport (or even removed from the DOM), it will remain the largest contentful element unless a larger element is rendered."

Then in an aside we can add the note:

"Prior to Chrome 88, removed elements were not considered as largest contentful elements, and removing the current candidate would trigger a new `largest-contentful-paint` entry to be dispatched. However, due to popular UI patterns such as image carousels that often removed DOM elements, the metric was updated to more accurately reflect what users experience. See the [CHANGELOG](https://chromium.googlesource.com/chromium/src/+/master/docs/speed/metrics_changelog/2020_11_lcp.md) for more details."

WDYT?

**Comment by philipwalton** [comment]
Also we can close this issue once this change is finalized: https://github.com/GoogleChrome/web.dev/issues/4457

**Pull request merged**